### PR TITLE
Resolve issues

### PR DESCRIPTION
**Pull request by orectique**
Hi @mmcky

The issues #16 , #17 , #18 , #19 , #20 have been resolved. 

The latest version can be found at https://6507c93430d1f5215b8a9c3d--poetic-sprite-e126a3.netlify.app/intro.

### CODE REVIEW

File: code-book/intro.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ kernelspec:
   name: julia-1.9
 ---
 
-# Dynamic Programming
+# Dynamic Programming <br/> Python and Julia Code
```
Comment (mmcky) [inline code-book/intro.md:12]: @orectique I think we can remove `Python and Julia Code` from the title as it is redundant (with information below) on the main page

File: code-book/_config.yml [hunk 2]
```diff
@@ -7,8 +8,14 @@ execute:
   execute_notebooks: "cache"
   timeout: 600 # 10 minutes
   allow_errors: true
+  exclude_patterns:
+    - '*.py'
+    - '*.jl'
+    - '*.ipynb' 
+
+only_build_toc_files: false
```
Comment (mmcky) [inline code-book/_config.yml:16]: It is usually after to leave the as `true`

then add `status` page to the `toc` to get it to build during compilation. 

I would suggest adding a third section `Other` then list `status` as a page.

**Review by mmcky — CHANGES_REQUESTED** [review]
@orectique nice work on this. Thanks for the fixes and updates. 

Only a few minor changes requested.

**Comment by orectique** [comment]
Apologies, @mmcky. I forgot to tag you after pushing the suggested edits.

**Review by mmcky — APPROVED** [review]

**Pull request merged**